src/ui/Header/Header.module.css

```diff
@@ -3,18 +3,19 @@
   top: 0;
   left: 0;
   right: 0;
-  height: 50px;
+  min-height: 50px;
   background-color: white;
   box-shadow: 0 0 2px 2px rgba(0, 0, 0, 0.1);
 
   display: flex;
-  justify-content: space-between;
-  align-items: center;
+  flex-direction: row;
+  flex-wrap: wrap;
   padding: 20px;
 }
 
 .logo {
   text-decoration: none;
   color: chocolate;
   font-weight: 900;
+  flex: 1;
```

Comment:
Let's shrink the logo on the desktop version to avoid misclicks?
<img width="1439" alt="«Screenshot» 2021-10-06 at 10 07 22" src="https://user-images.githubusercontent.com/9102374/136156202-8e1d7d9f-406f-4307-9dd8-69a8abf2fe77.png">